.github/workflows/pr.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: check out pr branch
         uses: actions/checkout@v2
       - name: Run tests in tester
-        uses: ./.github/actions/test
+        run: docker build -t uat ./; docker run -v /var/run/docker.sock:/var/run/docker.sock uat sbt clean test
```

Comment:
Question: I get -v mounts a volume, but what is `/var/run/docker.sock:/var/run/docker.sock`?

Reply:
Hehe!  This is a little bit of cleverness, explained in the wiki here: 
https://github.com/seattle-uat/universal-application-tool/wiki/Troubleshooting#the-tests-are-failing-since-docker-is-not-available-inside-the-container

Reply:
thanks!